nservicebus/sql-persistence/oracle-caveats.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Oracle Caveats
+component: SqlPersistence
+related:
+reviewed: 2017-04-06
+---
+
+The SQL Persistence provides autonomy between endpoints by using separate tables for every endpoint based on the endpoint name. However, due to Oracle's 30-character limit on table names and index names, the SQL Persistence must make some compromises.
+
+## Table Names
+
+NOTE: For a complete example of the schema created by the SQL Persistence for Oracle, see [Oracle Scripts](oracle-scripts.md).
+
+For storing subscriptions, timeouts, and outbox data, SQL Persistence will reserve 24 characters for the endpoint name, leaving 3 characters for the persistence type, and and additional 3 characters for an index type. Names are then constructed as `{EndpointName}{PersistenceTypeSuffix}{KeyType}`.
+
+The following table shows table names created for an endpoint named `My.Endpoint`:
+
+| Persistence type | Suffix |    Table Name    |       Indexes       |
+|------------------|:------:|:----------------:|:-------------------:|
+| Subscriptions    |  `_SS` | `MY_ENDPOINT_SS` | `MY_ENDPOINT_SS_PK` |
+| Timeouts         |  `_TO` | `MY_ENDPOINT_TO` | `MY_ENDPOINT_TO_PK`<br/>`MY_ENDPOINT_TK`<br/>`MY_ENDPOINT_SK` |
+| Outbox           |  `_OD` | `MY_ENDPOINT_OD` | `MY_ENDPOINT_OD_PK`<br/>`MY_ENDPOINT_IX` |
+
+If an endpoint name is longer than 24 characters, an exception will be thrown, and a substitute [table prefix](/nservicebus/sql-persistence/#installation-table-prefix) must be specified in the endpoint configuration:
+
+snippet: TablePrefix
+
+### Sagas
+
+Tables generated for sagas reserve 27 characters for the saga name, leaving 3 characters for the `_PK` suffix for the table's primary key.
+
+In order to accommodate as many characters for the saga name as possible, the [table prefix](/nservicebus/sql-persistence/#installation-table-prefix) is not represented in the resulting table name.
+
```

Comment:
> is not represented in the resulting table name

would it be simpler to say that table prefix is not supported for oracle or that it's ignored?